directives/workinghourspicker/working-hours-picker.directive.js

```diff
@@ -1,110 +1,118 @@
-(function() {
+(function () {
```

Comment:
The actual change of this file is:
![image](https://user-images.githubusercontent.com/7535597/35312562-06ecef4a-00f7-11e8-8640-d5e2bd1a2045.png)
This could be a remind that we should indent codes at beginning and format files in the same way.  Then it won't reflect a bunch of formatting changes when people change it and format it later.